lib/diagnostic.rb

```diff
@@ -5,36 +5,36 @@ class Diagnostic
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
-    # Your code goes here
+    list.max
```

Comment:
Almost. You need to use the spaceship comparison to see what is the largest.

`list.max { |a, b| a.length <=> b.length }`